model/riscv_addr_checks.sail

```diff
@@ -116,7 +116,7 @@ type ext_data_addr_error = unit
 
 /* Default data addr is just base register + immediate offset (may be zero).
    Extensions might override and add additional checks. */
-function ext_data_get_addr(base : regidx, offset : xlenbits, acc : AccessType(ext_access_type), width : word_width)
+function ext_data_get_addr(base : regidx, offset : xlenbits, acc : AccessType(ext_access_type), width : xlenbits)
```

Comment:
Is `xlenbits` the right type? Maybe `nat` would be better (or `range(1, 1024)` for performance reasons).